install/terraform/modules/gke/cluster.tf

```diff
@@ -35,6 +35,7 @@ locals {
   windowsInitialNodeCount = lookup(var.cluster, "windowsInitialNodeCount", "0")
   windowsMachineType      = lookup(var.cluster, "windowsMachineType", "e2-standard-4")
   autoscale               = lookup(var.cluster, "autoscale", false)
+  workload_id             = lookup(var.cluster, "workload_id", false)
```

Comment:
I recognize that `workload_id` matches the rest of the terraform universe, but our modules have been using camelCaps for a while, it seems. Additionally, I would go ahead and expand the shortening: `workloadIdentity`.